src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_NAME = 0;
+    private static final int INDEX_OF_NUM = 1;
+    private static final String VARIABLE_BUY = "buy";
+    private static final String VARIABLE_SUPPLY = "supply";
+    private static final String VARIABLE_RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(getReport(readInfoFromFile(fromFileName)),toFileName);
```

Comment:
```suggestion
        String inputData = readInfoFromFile(fromFileName);
        String report = getReport(inputData);
        writeToFile(report, toFileName);
```
давай зробимо твій код більш читабельним